### DISCUSSION TITLE
How to enable the sqliteConnection for PWA?

### DISCUSSION THREAD

**Discussion by nidhikachhawa24** [DJP-Digital-Jaaduii-Pitara/sakhi-api-service#121] [Q&A]
**Scenario:**

We are using eJaaduiPitara as a PWA. We want the local storage feature also, like **_create pitara_**, **_create playlist_** and list of **_recently Viewed contents_**, on the web app.

I checked that this connection of initialising the DB connection is not working:

```
this.sqlitePlugin = CapacitorSQLite;
this.sqliteConnection = new SQLiteConnection(this.sqlitePlugin);
```
_How can we enable this DB connection for web application?
How can we generalise so that this can work for android as well as web application also?_

cc: @swayangjit @kameshbhr

**Comment by swayangjit** [comment]
@nidhikachhawa24  As of now we are supporting only  app and we haven't tested the browser support as capacitor SQLITE has browser support you can check [this plugin](https://github.com/capacitor-community/sqlite) and let us know if you are facing any difficulties.

**Reply by kameshbhr** [reply]
>> How can we generalise so that this can work for android as well as web application also?

Generalization may not be required as they are two different apps. You can possibly go with a different approach all together.
Mobile app can use file storage as it would have access to it, whereas web app would only have access to browser cache, not the device filesystem.
